flow/project.py

```diff
@@ -638,15 +767,17 @@ def op2(job):
 
     MAX_LEN_ID = 100
 
-    def __init__(self, name, operations=None, operation_directives=None,
-                 options=""):
+    def __init__(self, name, flow_aggregate=None, operations=None,
```

Comment:
Ideally, the position of existing arguments shouldn't change. `flow_aggregate` should be last or second to last.